packages/SystemsThinkingWorkbench-Core.package/STWNode.class/instance/mouseDown..st

```diff
@@ -5,6 +5,6 @@ mouseDown: evt
 	self owner firstClicked
 		ifNotNilDo: [:node | node = self
 				ifTrue: [self owner firstClicked: nil]
-				ifFalse: [self createConnectionFrom: node.
+				ifFalse: [self createTransferConnectionFrom: node.
```

Comment:
See the respective comment on this method, maybe the `STWSimulation` should have a current tool instance variable which is the class of connections to create.